{{cookiecutter.project_slug}}/tutorial.ipynb

```diff
@@ -383,7 +383,6 @@
     "from art.steps import CheckLossOnInit\n",
     "from art.checks import CheckScoreCloseTo\n",
     "from torch.cuda import is_available\n",
```

Comment:
Please remove also the line importing is_available. I belive it is not used anywhere